### PR TITLE
Add reboot action to hub

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
*Description of changes:*
Add reboot Tinkerbell action image to hub

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/tinkerbell/hub/Makefile
Comment (jaxesn) [inline projects/tinkerbell/hub/Makefile:17]: Since there is no build for reboot, remove this.   I would just add this to image_names

`IMAGE_NAMES=$(BINARY_TARGET_FILES) reboot`

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (jaxesn) [inline projects/tinkerbell/hub/docker/linux/reboot/Dockerfile:9]: is touch a statically compiled binary?  if you run `ldd touch` does it show any dynamic libs?

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (jaxesn) [inline projects/tinkerbell/hub/docker/linux/reboot/Dockerfile:17]: I hate seeing the red circle, just add a new line.

**Review by jaxesn — CHANGES_REQUESTED** [review]

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (jaxesn) [inline projects/tinkerbell/hub/docker/linux/reboot/Dockerfile:15]: nit, but since touch is a standard system component, I would probably just put it in usr/bin to match where it would normally be.

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (abhinavmpandey08) [reply projects/tinkerbell/hub/docker/linux/reboot/Dockerfile:9]:
```
#7 7.768 + ldd ./usr/bin/touch
#7 7.772        linux-vdso.so.1 (0x00007ffe41b4f000)
#7 7.772        libc.so.6 => /lib64/libc.so.6 (0x00007f7f25c90000)
#7 7.772        /lib64/ld-linux-x86-64.so.2 (0x00007f7f2603b000)
#7 DONE 8.4s
```

File: projects/tinkerbell/hub/Makefile
Comment (abhinavmpandey08) [reply projects/tinkerbell/hub/Makefile]: done

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (abhinavmpandey08) [reply projects/tinkerbell/hub/docker/linux/reboot/Dockerfile]: done

**Review by jaxesn — CHANGES_REQUESTED** [review]
Is the CDK change done to create the new ecr repo for this?

**Review by jaxesn — APPROVED** [review]
This looks good to me. Holding until the cdk changes are through, @abhinavmpandey08 unhold when you are ready.

/lgtm
/hold

**Comment by abhinavmpandey08** [comment]
> This looks good to me. Holding until the cdk changes are through, @abhinavmpandey08 unhold when you are ready.
> 
> /lgtm /hold

will do that, thanks!

File: projects/tinkerbell/hub/docker/linux/reboot/Dockerfile
Comment (chrisdoherty4) [reply projects/tinkerbell/hub/docker/linux/reboot/Dockerfile:9]: Do we need flexibility over the base image? I ask because fiddling with libc versions can result in unrunnable binaries and letting maintainers twiddle that without knowing intricacies can result in confusion and a lot of time burnt.

I guess I'm wondering if its better to house configuration in a Makefile or, given the singular use of this image, just specify the FROM images as constants so there's no room for error.

**Pull request merged**
